scout/build/individual.py

```diff
@@ -83,12 +85,14 @@ def build_individual(ind):
     ind_files = [
         "bam_file",
         "mt_bam",
-        "vcf2cytosure",
+        "rna_coverage_bigwig",
+        "tiddit_coverage_wig",
         "rhocall_bed",
         "rhocall_wig",
-        "tiddit_coverage_wig",
```

Comment:
I kind of like alphabetical when the lists grow and there is no clear priority for ordering by file type, logic of use etc. But it works either way obviously.